fairroot/tools/CMakeLists.txt

```diff
@@ -20,6 +20,10 @@ set(sources
 
 fair_change_extensions_if_exists(.cxx .h FILES "${sources}" OUTVAR headers)
 
+list(APPEND headers
+  FairBaseTypes.h
```

Comment:
* How about just `FairTypes.h`?
* I would put such a header into the base lib instead of tools (which I interpret more as a collection of helpers/utilities)?